hildon/maemo-gtk-compat.h

```diff
@@ -0,0 +1,66 @@
+#ifndef MAEMOGTKCOMPAT_H
+#define MAEMOGTKCOMPAT_H
```

Comment:
Please, use "__MAEMOGTKCOMPAT_H__" in this file.
Try to keep the style of the package you are making changes to.

Reply:
fxied